docs/program/_index.md

```diff
@@ -3,8 +3,278 @@ title: "Program a Robot"
+  - "program/sdk-as-client"
+  - "program/sdks"
 ---
+
+Viam offers software development kits (SDKs) in popular languages which
+
+- Streamline connection, authentication, and encryption against a server using {{< glossary_tooltip term_id="webrtc" >}}
+- Enable you to interface with robots without calling the `viam-server` [gRPC APIs for robot controls](https://github.com/viamrobotics/api) directly
+
+![Diagram showing how a client connects to a robot with Viam. Diagram shows a client as a computer sending commands to a robot. Robot 1 then communicates with other robotic parts over gRPC and WebRTC and communicating that information back to the client.](img/sdks/robot-client.png)
+
+Use the SDK of your preferred language to write code to control your robots.
+
+Viam currently offers SDKs for the following languages:
+
+- [Python SDK](https://python.viam.dev/)
+- [Go SDK](https://pkg.go.dev/go.viam.com/rdk)
+- [TypeScript SDK](https://ts.viam.dev/)
+- [C++ SDK (alpha)](https://cpp.viam.dev/)
+
+## Requirements
+
+Before you get started with your program, ensure that you have [installed and connected to `viam-server`](/installation/) on the computer you want to use to control a robot [(likely a single-board computer)](/components/board/#configuration) and [configured a robot](/manage/configuration/#local-setup).
+
+The easiest way to do this is:
+
+1. Go to [app.viam.com](https://app.viam.com/).
+2. Create a new robot.
+3. Go to the **Setup** tab and follow the instructions there.
+4. [Configure](/manage/configuration/) your robot.
+
+Next, to install your preferred Viam SDK on your development machine, run one of the following commands in your terminal:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+pip install viam-sdk
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+go install go.viam.com/rdk/robot/client@latest
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+npm install --save @viamrobotics/sdk
+```
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Hello World: The Code Sample Tab
+
+Create a program in the language of your choice that connects to your robot and uses methods built into the SDK's client API libraries to [interact with and control](/program/sdks/) the {{< glossary_tooltip term_id="resource" text="resources" >}} on the robot.
+
+Start with this by navigating to your robot's page on [the Viam app](https://app.viam.com/robots).
+Select the **Code Sample** tab, select your preferred SDK, and copy the code generated for you.
+These code snippets import all the necessary libraries and set up a connection with your robot in the cloud through the Viam app.
```

Comment:
```suggestion
This code snippet imports all the necessary libraries to set up a connection with your robot and interface with its configured components and services.
```
Since your robot is not usually in the cloud and the generated code does create methods for all viam-server components and services (not modular ones at the moment)